packages/web-twig/tests/__snapshots__/ComponentsSnapshotTest__test with data set dropdownDefault.twig__1.html

```diff
@@ -1,8 +1,13 @@
 <html><body>
```

Comment:
Updated snapshot of the Dropdown should not be part of this PR.

Reply:
my bad, sorry :)